ramls/blocked_fields.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Why does this filename use an underscore rather than a hyphen? 

Maybe it is because two very old schema (that I created) use underscores. These should really be renamed (though they have both been deprecated for well over a year)

Reply:
Sure, I could rename that schemas as well